packages/generator/src/generator.ts

```diff
@@ -37,6 +35,8 @@ abstract class Generator<T extends GeneratorOptions = GeneratorOptions> extends
   private performedActions: string[] = []
   private useTs: boolean
 
+  abstract sourceRoot: string
```

Comment:
we had to make this change since templates are now relative to the generator rather than the `generate` CLI command.